python/iiboost/booster.py

```diff
@@ -22,7 +22,8 @@
 import numpy as np
 import ctypes
 
-from exceptions import RuntimeError
+if sys.version_info[0] < 3:
+    from exceptions import RuntimeError
```

Comment:
Also might be worth adding `from __future__ import print_function` here as well.